### PR TITLE
Display issue checks history graph

### LINKED ISSUE #320: Add endpoint + graph visualization
Should be easy to make a graph like on coverage front to list the number of issues detected per day

### PR DESCRIPTION
**Pull request by La0**
Fixes #320

### CODE REVIEW

**Comment by La0** [comment]
Runs on testing (as it needs the backend to be updated too) : https://code-review.stage.mozaws.net/#/stats

File: backend/code_review_backend/issues/api.py [hunk 1]
```diff
@@ -154,6 +156,48 @@ class IssueCheckStats(generics.ListAPIView):
...
+    * globally
+    * per repository
+    * per analyzer
+    * per check
+    """
+
+    serializer_class = HistoryPointSerializer
+
+    # For ease of use, the history is available without pagination
+    # as the SQL request should always fast to calculate
```
Comment (marco-c) [inline backend/code_review_backend/issues/api.py]:
```suggestion
    # as the SQL request should be always fast to calculate
```

**Comment by marco-c** [comment]
> Runs on testing (as it needs the backend to be updated too) : https://code-review.stage.mozaws.net/#/stats

Why is the first point in the graph so high?

**Comment by La0** [comment]
Because i imported the existing issues from Taskcluster index on the first day

**Pull request merged**